languages/dart/keywords/abstract.md

```diff
@@ -0,0 +1,83 @@
+# Abstract
+
+The keyword `abstract` is used to define an *abstract class* - a class that can not be instantiate directly, which means
+you can not create an object from it. Conversely, a class that can be instantiated directly is called a *concrete class*.
+
+Abstract types are interfaces that hide the implementation details and show only functionality to the user. 
+They let you focus on **what** the object does instead of **how** it does it.
+
+```dart
+// This class is declared abstract and thus can't be instantiated.
+abstract class CoffeeMachine {
+  //Body of abstract class
+}
+
+CoffeeMachine coffeeMachine = CoffeeMachine(); // Abstract classes can't be instantiated.
+```
+
+An abstract class can have constructors (including factories), fields, methods as well as static members. 
```

Comment:
Do I have to split `static` and `member` ? In this case, it is only one notion (more about `static` then `members`) and I think this part will be covered in the `static` definition.  What do you think ?

Reply:
I think it should not be split. I also think `members` should be common enough concept in other languages that it doesn't need it's own reference. So if there exists a common reference to `members` it should be linked, otherwise left as is.

Reply:
`static` in Dart's context should have it's own dart specific reference file. Since static is a concept that varies in it's implementation as well as interpretation language to language.

Reply:
There is a  file `class` with a part `fields` in Dart's keyword folder. May be I can link `members` to it ?

Reply:
Sounds good to me